worlds/pokemon_rb/__init__.py

```diff
@@ -703,6 +703,7 @@ def fill_slot_data(self) -> dict:
             "require_pokedex": self.options.require_pokedex.value,
             "area_1_to_1_mapping": self.options.area_1_to_1_mapping.value,
             "blind_trainers": self.options.blind_trainers.value,
+            "v5_update": True,
```

Comment:
```suggestion
            "pre_v5": False,
```
This way, it'll be easier to safely remove this further down the line, since in a few months there will be no point to supporting the v4 logic.

Reply:
I really don't see the point of the change tbh. It will take months for it to remove since it can only be removed with an AP update. It's not hurtful to have this in longer.

Reply:
It's a difference between checking null or yes and checking for an explicit no.

The way you're doing it currently, you'd check for v5_logic == false or v5_logic == nil when you could be checking for pre_v5 logic == false.

Reply:
At some point down the line, you'll have an ecosystem where some people are using an apworld with that piece of data in slot data and some that don't, and everyone will have a poptracker that checks for that data to see whether to use v4 or v5 logic.
Unless you just remove the v4 logic entirely from the slot data, then it's whatever. Will break stuff for people who haven't updated their poptracker unnecessarily, but that's not a big deal.

Reply:
https://discord.com/channels/731205301247803413/731214280439103580/1300955726302285846

an hour long convi in #ap-core-dev. People have lost their minds and are unsure what even is real anymore. Will keep this as is.